### DISCUSSION TITLE
keyball61でTRRSケーブルで接続すると片方しか入力できない

### DISCUSSION THREAD

**Discussion by kentakov** [Yowkees/keyball#251] [Q&A]
keyball61を組み立て終わりました。
片方ずつ接続すると入力が可能なのですが。
TRRSケーブルで接続するとUSBで接続していない側からの入力ができない状態です。
![IMG_4172](https://github.com/Yowkees/keyball/assets/136250582/bc7e06f0-2596-4e67-b7b3-a7962d2b9f2a)

基盤の状態は以下
![IMG_4173](https://github.com/Yowkees/keyball/assets/136250582/a835d644-5488-4822-9edf-70b71883209d)
![IMG_4174](https://github.com/Yowkees/keyball/assets/136250582/098b08e5-df6f-409c-bdb1-f578858bd2d6)

組み立ての途中でピンソケットからコンスル―に変更した際に左手側表の赤丸部分のランドを欠損してしまっています。
こちらが原因でしょうか？
![IMG_4116](https://github.com/Yowkees/keyball/assets/136250582/5fab9558-45b0-4ef2-90a2-490dd2c0758f)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ご指摘通り、ランドの破損個所が左右通信の端子ですので、それが原因かと思います。

この修正方法ですが、下記写真の緑点２点をエナメル線やゴム被覆導線で接続してください。
もしコンスルーに半田付けしたくないということでしたら、ProMicro基板に直接配線していただければと思います。
お手数ですがご対応お願いいたします。

![KB61L_fix1](https://github.com/Yowkees/keyball/assets/84209772/c1abc137-1a5d-4a11-991e-2ede0f131ffe)

**Reply by kentakov** [reply]
![image](https://github.com/Yowkees/keyball/assets/136250582/71a238e7-d4a9-419e-876c-a54a6ee76d6f)
解説ありがとうございます！無事に配線が完了して動作確認ができました！！本当にありがとうございます。

**Comment by Yowkees** [comment]
無事に直ったようでよかったです！
